docs/reference/commandline/network_ls.md

```diff
@@ -51,11 +51,25 @@ Multiple filter flags are combined as an `OR` filter. For example,
 
 The currently supported filters are:
 
+* driver
 * id (network's id)
 * label (`label=<key>` or `label=<key>=<value>`)
 * name (network's name)
 * type (custom|builtin)
 
+#### Driver
+
+The `driver` filter matches networks based on their driver.
+
+The following example matches networks with the `bridge` driver:
+
+```bash
+$ docker network ls filter driver=bridge
```

Comment:
oh! thanks for catching that; I'll update when I'm at my computer (public holiday today in the Netherlands)